src/options.h

```diff
@@ -6,15 +6,17 @@
 #include "config_parser.h"
 
 struct Options {
-	unsigned short port;
-	std::map<std::string, RequestHandler*> handlerMap;
-	RequestHandler* defaultHandler;
-	bool addHandler(std::string type, std::string path, const NginxConfig& handler_config);
-	bool addPath(std::shared_ptr<NginxConfigStatement> handler_config);
-	bool addDefaultHandler(std::shared_ptr<NginxConfigStatement> handler_config);
-	bool addPort(std::shared_ptr<NginxConfigStatement> port_config);
-	bool loadOptionsFromStream(std::istream* config_file);
-	bool loadOptionsFromFile(const char* filename);
-	~Options();
+    unsigned short port;
+    unsigned short thread;
```

Comment:
I know the limit is set to 2000, but I don't think this should necessarily be a `short`. Maybe just `unsigned int` or `size_t`?